src/app/containers/CpsOnwardJourney/index.jsx

```diff
@@ -12,18 +11,73 @@ import {
   GEL_SPACING_DBL,
   GEL_SPACING_TRPL,
 } from '@bbc/gel-foundations/spacings';
-
+import Grid from '@bbc/psammead-grid';
 import { storyItem } from '#models/propTypes/storyItem';
 import { ServiceContext } from '#contexts/ServiceContext';
 import { GridWrapper, GridItemConstrainedLarge } from '#lib/styledGrid';
+import { gelGridMargin } from '#app/lib/layoutGrid';
+
+const ConstrainedLargeGrid = ({ children, ...gridProps }) => (
+  <Grid
+    item
+    {...gridProps}
+    columns={{
+      group0: 1,
+      group1: 1,
+      group2: 1,
+      group3: 1,
+      group4: 6,
+      group5: 12,
+    }}
+    margins={{
+      group0: true,
+      group1: true,
+      group2: true,
+      group3: true,
+      group4: false,
+      group5: false,
+    }}
+    startOffset={{
+      group0: 1,
+      group1: 1,
+      group2: 1,
+      group3: 1,
+      group4: 3,
+      group5: 6,
+    }}
+    parentColumns={{
+      group0: 8,
+      group1: 8,
+      group2: 8,
+      group3: 8,
+      group4: 4,
+      group5: 3,
+    }}
```

Comment:
Where do these values come from? Are they configured in the MAP page and Story page? If so the docs should instruct that this component must be used in a parent grid with this exact column configuration.

Reply:
I think @rhenshaw56 did it by how it looked and comparison; I'm not sure if psammead-grid is in use in those components yet, but will check.

Reply:
Ah okay, I didn't think you'd need parent grid config, if there isn't a parent grid 🤔

Reply:
👀   @andrewscfc there is a parent grid config defined [here](https://github.com/bbc/simorgh/blob/latest/src/app/pages/StoryPage/index.jsx#L199-L205) which encompasses this as the [second column](https://github.com/bbc/simorgh/blob/latest/src/app/pages/StoryPage/index.jsx#L244-L248)

Reply:
Ah okay, that makes more sense @rhenshaw56 , I do notice that `group5` is `3` in this component but is `4` in the Story page - I'd expect them to be the same? 

The reason I'm picking up on this is there is an implicit coupling between the Story page and this component; the column configuration of the Story page could change causing a regression. I said above to document this but maybe it would be better to pass the parentColumns like in this example for frontpage:
https://github.com/bbc/simorgh/blob/74221208cd41fc4f6c63aa1ecbec0b75a2794f92/src/app/containers/FrontPageStoryRows/index.jsx#L62 

This would make the coupling explicit so future maintainers will be aware they need to modify `CpsOnwardJourney` when they modify the secondary column config in the Story page.